src/components/AudioPlayer/PlayerButtons/Forward/ForwardButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import PlayerButton from '../PlayerButton';
+
+import { playNextSong } from 'store/actions/playerActions';
+import { canPlayNextSong } from 'store/selectors/songsQueue';
+
+import fastForwardIcon from '../../icons/fast-forward.svg';
+
+const ForwardButton = () => {
+  const onRepeat = useSelector((state) => state.player.get('onRepeat'));
+  const canFastForward = useSelector((state) => canPlayNextSong(state));
+
+  const isDisabled = !onRepeat && !canFastForward;
+
+  const classStatus = isDisabled ? 'disable-buttons' : '';
+  const dispatch = useDispatch();
+
+  return (
+    <PlayerButton
+      classStatus={classStatus}
```

Comment:
Why don't you just use `className`? So people don't have to check your component's implementation to know what this prop is about.